src/containers/audio-rewards-page/config.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { ReactNode } from 'react'
+
+import { ChallengeRewardID, TrendingRewardID } from './types'
+
+type ChallengeRewardsInfo = {
+  id: ChallengeRewardID
+  title: string
+  icon: ReactNode
+  description: string
+  fullDescription: string
+  progressLabel: string
+  amount: number
+  stepCount: number
+  buttonText: string
+}
+
+export const challengeRewardsConfig: Record<
+  ChallengeRewardID,
+  ChallengeRewardsInfo
+> = {
+  'invite-friends': {
+    id: 'invite-friends' as const,
```

Comment:
if the id is always the same as the key do we really need it? I can see the use case for it to be easier to pass around. 
Should this is `as ChallengeRewardID` instead of `as const`?